addons/mail/static/src/messaging/entity/locale/locale.js

```diff
@@ -0,0 +1,39 @@
+odoo.define('mail.messaging.entity.Locale', function (require) {
+'use strict';
+
+const { registerNewEntity } = require('mail.messaging.entity.core');
+
+function LocaleFactory({ Entity }) {
+
+    class Locale extends Entity {
```

Comment:
Still not sure about the necessity of this. Having different parts of the interface in different local under the same env seems far fetched.

As long as this doesn't have anything in the store, this is pretty much useless and just an additional step as opposed to just having `textDirection` in env.

I don't mind if we keep it for now, but if you somewhat agree I feel like we should remove it.

Reply:
I think we should keep modelling this entity, even if it does almost nothing for the moment.

in our channel R&D Tasks, yesterday or so, I rapidly read a discussion in which translation should only be applied in the content of the app store content part of the page, and not on the whole page. I really think this entity will make a lot of sense the more we refactor odoo codebase.

Reply:
But `_t` will be wrong too then. I think having in mind the feature is good, but even then it seems it should be at `env` level and not entity level.

The different elements that will have a different locale will need different `env`.